test/testgeometry.c

```diff
@@ -27,7 +27,7 @@ static SDLTest_CommonState *state;
 static SDL_bool use_texture = SDL_FALSE;
 static SDL_Texture **sprites;
 static SDL_BlendMode blendMode = SDL_BLENDMODE_NONE;
-static double angle = 0.0;
+static float angle = 0.0;
```

Comment:
While you are there, change value to `0.0f` ?

Reply:
I just did.